extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -334,12 +352,62 @@ By default, this attribute is not present.
 
 ##### src [optional]
 
-If present, `amp-date-picker` requests JSON data to populate the
-`highlighted` and `blocked` lists, as well as matching templates in the document
-to lists of dates.
+If present, `amp-date-picker` requests JSON data to populate certain attributes
+dynamically, as well as matching lists of dates to template `id`s for
+rendering days in the calendar.
+
+###### blocked
+
+A list of dates to render as blocked in the calendar view. The user is
+prevented from selecting these dates.
+
+###### date
+
+Specifies the initially selected date.
+In a date picker with this has no effect.
```

Comment:
Clarify the second sentence as I'm confused as to what it means.  The whole thing is about a date picker, so how it can be no effect?

Reply:
Oops I accidentally a word. Should say "In a date range picker this has no effect."